_posts/2020-11-30-developer-survey-2020.md

```diff
@@ -0,0 +1,99 @@
+
+Compared to results from 2018, JavaScript fell down from number #2 spot to number #4. This could be explained TypeScript, which gained almost as many percents now than JavaScript lost. All in all, most top languages were in decline compared to the previous survey, except Kotlin, which received much more love this year compared to 2018. This time there also seemed to be a lot more niche languages getting a few votes than before, which also explains why the top languages were a bit in decline.
+
+## Languages we would like to use
+![Languages we would like to use](/img/developer-survey-2020/would_like_to_use_languages.png)
+
+The answers to this question tells us several things. First, it tells us that for widely already used languages there is still interest in continuing to use them. Second, the answers can tell us that there's even more interest in some languages, so their usage could still grow in the future. Third, for languages that are not yet widely used, the answers tell us what languages we could be potentially be using in the future.
+
+Kotlin was the most wanted language here, with 29% of the respondents saying they would like to use Kotlin in the future. As such, people using Kotlin currently seem to want to continue using it and others would like to start using it. As such, Kotlin seems like a good choice for future projects.
+
+Clojure came third, with 22% wanting to work with it in future. TypeScript came third with 18%. With TypeScript usage having increased a lot in two years, the interest is still growing.
+
+For languages not currently in wide use, Rust was a clear winner, with 8% of respondents wanting to use it. With appropriate projects, we could very well see Rust move to the used languages category for the next survey.
+
+## Frontend libraries and frameworks
+![Frontend frameworks](/img/developer-survey-2020/frontend_frameworks_and_libraries.png)
+
+React rules the frontend at the moment. 66% who answered to the question about used frontend libraries or frameworks reported having used React in the last year. All the competion (Angular, Aurelia, Vue) combined actually only just barely reached React's numbers. Over half of the React users also used Redux with it and a half also used React Router. ClojureScript developers rely on Reagent as their React library and then most of them also use re-frame for state-management.
+
+For ready-made styles or React components, about a fifth of the answerers reported using Bootstrap or Material-UI.
```

Comment:
Ready-made styles could be just CSS frameworks.

Reply:
Fixed